.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+yarnPath: .yarn/releases/yarn-4.5.1.cjs
+enableScripts: false
```

Comment:
Docs for this are here: https://yarnpkg.com/configuration/yarnrc#enableScripts. We don't want to run post-install scripts from third party libraries, and this config replaces `--ignore-scripts` from Yarn Classic.